community.md

```diff
@@ -2,54 +2,61 @@
-* [Blazor Chat sample](https://github.com/conficient/BlazorChatSample) ([Demo](https://blazorchatsample.azurewebsites.net/)) &ndash; Uses SignalR via JS interop.
-* [Blazor Clock Component](https://github.com/Lupusa87/LupusaBlazorProjects) ([Demo](http://lupusablazordemos.azurewebsites.net/))
-* [Blazor Graph API example](https://github.com/jburman/BlazorGraphExample) ([Demo](https://blazorgraph.z20.web.core.windows.net/))
-* [Blazor Hacker News Clone](https://github.com/lohithgn/blazor-hackernews-clone)
-* [Blazor RealWorld example app](https://github.com/torhovland/blazor-realworld-example-app)
-* [Blazor Tour of Heroes](https://github.com/lohithgn/blazor-tour-of-heroes)
-* [Blazor To-Do App](https://github.com/BorowskiKamil/blazor-tasks)
+* [Blazor + Electron sample](https://github.com/SteveSandersonMS/BlazorElectronExperiment.Sample) &ndash; Explore how a Blazor app can be used to build a cross-platform desktop app.
+* [Blazor + Sitecore](https://github.com/GoranHalvarsson/SitecoreBlazor) ([Blog post](https://visionsincode.wordpress.com/2018/06/30/time-travel-into-the-future-blazor-sitecore-helix/)) &ndash; Example of dynamic pages and routes with SiteCore and [Helix](https://helix.sitecore.net/).
+* [Blazor Bricks](https://www.codeproject.com/Articles/1241210/WebAssembly-with-Blazor) &ndash; A bricks game developed with C# and Blazor.
+* [Blazor Chat sample](https://github.com/conficient/BlazorChatSample) ([Demo](https://blazorchatsample.azurewebsites.net/)) &ndash; An app that demonstrates the use of [SignalR](https://docs.microsoft.com/aspnet/core/signalr/) to create a Blazor chat app.
+* [Blazor Graph API example](https://github.com/jburman/BlazorGraphExample) ([Demo](https://blazorgraph.z20.web.core.windows.net/)) &ndash; A self-contained Blazor app that connects to [Microsoft's Graph API](https://docs.microsoft.com/azure/active-directory/develop/active-directory-graph-api), queries the user's information, and allows the user to browse through their [OneDrive](https://onedrive.live.com/about/).
+* [Blazor Hacker News Clone](https://github.com/lohithgn/blazor-hackernews-clone) &ndash; A Hacker News clone built in Blazor.
+* [Blazor RealWorld example app](https://github.com/torhovland/blazor-realworld-example-app) &ndash; A full-stack app built with Blazor that includes CRUD operations, authentication, routing, and pagination.
+* [Blazor Tour of Heroes](https://github.com/lohithgn/blazor-tour-of-heroes) &ndash; A Blazor implementation of [Angular Tour of Heroes](https://angular.io/tutorial).
+* [Blazor To-Do App](https://github.com/BorowskiKamil/blazor-tasks) &ndash; A Blazor app that allows you to create and edit tasks and manage categories of tasks.
 * [Blazor UWP Sample](https://github.com/pushqrdx/Blazor.Universal) &ndash; Use Blazor.Server to build UWP apps.
-* [Toss](https://github.com/RemiBou/Toss.Blazor)
+* [Lupusa Demos](https://github.com/Lupusa87) ([Demos](https://lupusablazordemos.azurewebsites.net/))
```

Comment:
I'd prefer not to have a top level grouping like this. Let's keep the list flat.

Reply:
I would keep all components in a single list for now. We don't have _that_ much of them that create categories makes sense like in awesome-xxx repos, so for now, a simple list would keep it simpler. Also at this page, or perhaps another one, I think it would worth to have something like we did in Orleans http://dotnet.github.io/orleans/Documentation/who_is_using_orleans.html so people can put their companies there.